tools/truffle-example/test/greeter.js

```diff
@@ -0,0 +1,47 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Truffle Example
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const Greeter = artifacts.require('Greeter');
+
+contract('Greeter', () => {
+  let contractInstance;
+  const initialMsg = 'initial_msg';
+  const updatedMsg = 'updated_msg';
+
+  it('should be able to deploy a contract', async () => {
```

Comment:
Should be similar to other tools examples where there's an HBAR transfer example also.